app/build.gradle

```diff
@@ -43,19 +48,45 @@ android {
             excludes += '/META-INF/{AL2.0,LGPL2.1}'
         }
     }
+    viewBinding {
+        enabled true
+    }
 }
 
 dependencies {
     implementation 'androidx.core:core-ktx:1.7.0'
```

Comment:
What about include the navigation dependency?

Reply:
I think it is a simple feature integration and one single fragment, hence integrate navigation component will increase complexity. Will be worth if has more than one fragments

Reply:
Ok, we can add it when we need use it

Reply:
I am not very fond of the navigation library.
Anyway, we can start using it, if we would need it.